radareorg/cutter
# Do not open multiple setting dialogs to fix #2314

**Your checklist for this pull request**



**Detailed description**

This pull-request fixes #2314. Setting dialogs are now not opened repeatedly.

**Test plan (required)**

1. Edit -> Preferences
2. Edit -> Preferences
3. Check two settings dialogs are not opened.

**Closing issues**

"closes #2314"
